revive.toml

```diff
@@ -21,6 +20,8 @@ warningCode = 0
 [rule.modifies-parameter]
 
 # Add these once issues are fixed
+[rule.var-naming]
+  severity = "warning"
```

Comment:
A lot of places violates rule.var-naming, it makes the PR too complicated to fix all.

I decide to change the error level of it to "warning" temporarily. We can fix it in a separate PR.